content/news/051/index.md

```diff
@@ -279,32 +313,32 @@ Special mention to the following talks, relevant to game development:
 [rustnl_michael_melchiore_talk]: https://www.youtube.com/watch?v=XLefuzE-ABU&t=3898s
 [rustnl_bevy_engine]:https://bevyengine.org/
 
-## Meeting Minutes
-
-<!-- Up to 10 most important notes + a link to the full details -->
-
-[See all meeting issues][label_meeting] including full text notes
-or [join the next meeting][join].
+## Misc. Links
 
-[label_meeting]: https://github.com/rust-gamedev/wg/issues?q=label%3Ameeting
+### New libraries
 
-## Discussions
+- [egui_ratatui](https://github.com/gold-silver-copper/egui_ratatui): A ratatui backend that is also an egui widget. 
+Deploy on the web with WASM or ship natively with Bevy, macroquad, or eframe 
+- [bevy_ratatui_render](https://github.com/cxreiff/bevy_ratatui_render): A Bevy plugin for rendering your game to the terminal using ratatui
+- [bevy_light_2d](https://github.com/jgayfer/bevy_light_2d):  General purpose 2D lighting for the Bevy game engine. 
 
-<!-- Links to handpicked reddit/twitter/urlo/etc threads that provide
-useful information -->
+### Library Updates
 
-## Requests for Contribution
+- [Kira 0.9](https://github.com/tesselode/kira/releases/tag/v0.9.0): Library for expressive game audio
+- [Quinn 0.11.2](https://github.com/quinn-rs/quinn/releases/tag/0.11.2): Async-friendly QUIC implementation in Rust 
+- [rolt and jolt-sys 0.3.1](https://github.com/SecondHalfGames/jolt-rust/blob/main/CHANGELOG.md#v030):  Rust bindings to Jolt Physics 
+- [Hanabi 0.11](https://github.com/djeedai/bevy_hanabi/blob/v0.11.0/CHANGELOG.md): A GPU particle system plugin for the Bevy game engine.
 
-<!-- Links to "good first issue"-labels or direct links to specific tasks -->
+### Blog Posts
 
-## Jobs
-
-<!-- An optional section for new jobs related to Rust gamedev -->
+- [Architectural Ruminations][pixel-wizards-arch]: reflections on using Rust for game development,
+ prompted by LogLog Games's _[Leaving Rust game dev after 3 years]_.
```

Comment:
I'm not totally opposed to this, but it's listed in 2 places (the other being in pixel-wizards section)

Reply:
You're right. Fixed.